src/Stratis.Bitcoin.Features.BlockStore/AddressIndexing/AddressIndexer.cs

```diff
@@ -535,6 +556,37 @@ public AddressBalancesResult GetAddressBalances(string[] addresses, int minConfi
             }
         }
 
+        /// <inheritdoc />
+        public VerboseAddressBalancesResult GetVerboseAddressBalancesData(string[] addresses)
```

Comment:
Perhaps look at combining (refactoring) `GetAddressBalances` and this method so that there is some re-use of code.

Reply:
Also thought about it but there are hardly 2 pieces of code except for checks at the beginning of the methods that are the same.